flake/dev/all-maintainers.nix

```diff
@@ -0,0 +1,64 @@
+{ lib, ... }:
+{
+  perSystem =
+    { pkgs, config, ... }:
+    {
+      apps.all-maintainers = {
+        program = pkgs.writeShellApplication {
+          name = "update-all-maintainers";
+          runtimeInputs = with pkgs; [
+            git
+            coreutils
+          ];
+          text = ''
+            root="$(git rev-parse --show-toplevel)"
+            install -m 644 -T \
+              ${config.packages.all-maintainers} \
+              "$root/stylix/generated/all-maintainers.nix"
+          '';
+        };
+        meta.description = "update all-maintainers.nix";
+      };
+
+      packages.all-maintainers =
+        let
+          meta = pkgs.callPackage ../../stylix/meta.nix { };
+        in
+        pkgs.runCommand "all-maintainers"
+          {
+            passAsFile = [ "maintainers" ];
+            maintainers = lib.pipe meta [
+              lib.attrsToList
+              (map ({ value, ... }: value.maintainers or [ ]))
+              builtins.concatLists
+              (map (value: {
+                name = value.github;
+                inherit value;
+              }))
+              builtins.listToAttrs
+              (lib.generators.toPretty { })
+            ];
+            inherit (config.treefmt) projectRootFile;
+            nativeBuildInputs = [ config.formatter ];
+          }
+          ''
+            touch "$projectRootFile"
+
+            {
+              echo "# THIS FILE IS GENERATED, DO NOT EDIT"
+              # TODO: print the rest of the header
```

Comment:
The only reason I had a TODO in my patch set is because I couldn't find the old commits that you force-pushed over that contained the previous file header. And for some reason, I though you had a longer (multiline) header, but looking at [`537827f4`](https://github.com/nix-community/stylix/blob/537827f42846266f6c5a0ed8675aadb27d22daab/stylix/generated/all-maintainers.jsonc), it seems I was wrong.

You don't need this TODO if you're happy with the current header. If you're not happy with the current header, that should be fixed in this PR instead of having this TODO.

One thing that may be worth having in the header is _how_ the file is generated; i.e. `nix run .#all-maintainers`